amida-tech/indaba-client
# Feature/inba 101 project reorg

#### What's this PR do?
Creates a new `projectReducer` in the commons, reducing the weight on the workflow matrix UI. This is technically the very last of the commons reducers we should need for all foreseeable updates. It also reduces some of the complexity of the workflow matrix by use of `bindActionCreators`. 

#### Related JIRA tickets:
Technically... all of them.

#### How should this be manually tested?
Thoroughly check everything. Assign tasks, create projects, turn survey and projects on and off, add stages and subjects. If anything breaks, bug me immediately. 

#### Any background context you want to provide?

#### Screenshots (if appropriate):
